vipps-login-api.md

```diff
@@ -2,7 +2,7 @@
 
 API version: 1.0  
 
-Document version 2.0.2.
+Document version 3.0.0.
```

Comment:
Bumping the major version suggest that there is some breaking change.
I don't think the versioning gives much value anyway, so we might as well remove it
```suggestion
Document version 2.0.3
```